vulture_os/toolkit/api_parser/proofpoint_tap/proofpoint_tap.py

```diff
@@ -213,18 +211,15 @@ def execute(self):
                     except ProofpointTAPAPIError as e:
                         error = "API error: {}".format(e)
                 else:
-                    error = "{}".format(e)
+                    error = "{}".format(e) # should be fixed here IMPORTANT
```

Comment:
```suggestion
                    error = "{}".format(e)
```

Reply:
I have no idea why I put that to begin with, I will look into it

Reply:
Corrected